src/devices/machine/ibmps2.cpp

```diff
@@ -0,0 +1,1113 @@
+// license:BSD-3-Clause
+// copyright-holders:Wilbert Pol, Miodrag Milanovic, Carl
+/***************************************************************************
+
+    IBM PS2 Compatibles
+
+***************************************************************************/
+
+#include "emu.h"
+#include "bus/rs232/rs232.h"
+#include "bus/rs232/hlemouse.h"
+#include "bus/rs232/null_modem.h"
+#include "bus/rs232/rs232.h"
+#include "bus/rs232/sun_kbd.h"
+#include "bus/rs232/terminal.h"
+#include "bus/mca/mca.h"
+#include "bus/mca/mca_cards.h"
+#include "bus/mca/planar_uart.h"
+#include "bus/mca/planar_lpt.h"
+#include "bus/mca/planar_fdc.h"
+#include "bus/mca/planar_vga.h"
+#include "imagedev/floppy.h"
+#include "machine/ibmps2.h"
+#include "cpu/i86/i286.h"
+#include "cpu/i386/i386.h"
+#include "formats/pc_dsk.h"
+#include "softlist_dev.h"
+#include "speaker.h"
+#include "xtal.h"
+
+#define LOG_PORT80  0
+
+#define LOGSYSPORTS   	(1U <<  2)
+#define LOGNVRAM		(1U << 	3)
+#define LOGTIMERS		(1U <<	4)
+#define LOGPOST			(1U <<	5)
+#define LOGPOS			(1U <<	6)
+
+#define VERBOSE (LOGSYSPORTS|LOGNVRAM|LOGTIMERS|LOGPOST|LOGPOS)
+#include "logmacro.h"
+
+#define LOG_SYSPORTS(...) 	LOGMASKED(LOGSYSPORTS, __VA_ARGS__)
+#define LOG_NVRAM(...) 		LOGMASKED(LOGNVRAM, __VA_ARGS__)
+#define LOG_TIMERS(...)		LOGMASKED(LOGTIMERS, __VA_ARGS__)
+#define LOG_POST(...)		LOGMASKED(LOGPOST, __VA_ARGS__)
+#define LOG_POS(...)		LOGMASKED(LOGPOS, __VA_ARGS__)
```

Comment:
This is the opposite to the way everything else defines macros.  Everything else uses `LOG_FOO` for the constants, and `LOGFOO` for the function-like macros.